runtime/vm/jvminit.c

```diff
@@ -2297,6 +2297,7 @@ VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved)
 
 			/* Enable -XX:+UseContainerSupport by default */
 			if (argIndex >= argIndex2) {
+				vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_USE_CONTAINER_SUPPORT;
 				uint64_t subsystemsEnabled = omrsysinfo_cgroup_enable_subsystems(OMR_CGROUP_SUBSYSTEM_ALL);
```

Comment:
This is not strictly legal C code: the declaration must occur before the assignment.
@babsingh Please create a new pull request to correct this.

Reply:
Fix created: https://github.com/eclipse-openj9/openj9/pull/16526.